bin/packages/get-babel-config.js

```diff
@@ -19,6 +19,14 @@ const plugins = map( babelDefaultConfig.plugins, ( plugin ) => {
 	return plugin;
 } );
 
+if ( process.env.TRANSFORM_JSX_PRAGMA ) {
```

Comment:
Also not sure why we need all these env variables (this one and the include/exclude ones)
Can't we just always use this plugin? What's the issue if always include it?

Reply:
See https://github.com/WordPress/gutenberg/pull/7493#issuecomment-399548925

Reply:
I see so I guess this would solve the issue right? #7556